src/Core/Rendering/D3D12/Swapchain.cpp

```diff
@@ -4,8 +4,14 @@
 #include <Core/Utils/Log.h>
 
 #include <Core/Rendering/D3D12/CommandList.h>
+#include <Core/Window/Window.h>
+#include <DirectXMath.h>
+#include <functional>
 #include "GPUDevice.h"
 #include "DXGIFactory.h"
+#include <dxgiformat.h>
```

Comment:
These includes are either not needed, can be added to the D3D12SDK.h or are already inherited from other includes.